shared/notifications/index.js

```diff
@@ -0,0 +1,94 @@
+import { getSwapsFeatureLiveness } from '../../ui/app/ducks/swaps/swaps';
+import { getSwapsEthToken, getIsMainnet } from '../../ui/app/selectors';
+import { BUILD_QUOTE_ROUTE } from '../../ui/app/helpers/constants/routes';
+
+export const UI_NOTIFICATIONS = {
+  1: {
+    id: 1,
+    title: 'Now Swap tokens directly in your wallet!',
+    description:
+      'MetaMask now aggregates multiple decentralized exchange aggregators to ensure you always get the best swap price with the lowest netwrok fees.',
+    date: '02/22/2020',
+    image: 'images/swaps-logos-small.svg',
+    actionText: 'Start swapping',
+  },
+  2: {
+    id: 2,
+    title: 'MetaMask Mobile is here!',
+    description:
+      'Sync with your extension wallet in seconds. Scan the QR code with your phone camera to download the app.',
+    date: '02/22/2020',
+    actionText: 'Get the mobile app',
+  },
+  3: {
+    id: 3,
+    title: 'Help improve MetaMask',
+    description: 'Please share your experience in this 5 minute survey.',
+    date: '02/22/2020',
+    actionText: 'Start survey',
+  },
+};
+
+function getNotificationFilters(state) {
+  const currentNetworkIsMainnet = getIsMainnet(state);
+  const swapsIsEnabled = getSwapsFeatureLiveness(state);
+
+  return {
+    1: !currentNetworkIsMainnet || !swapsIsEnabled,
+  };
+}
+
+export function getSortedNotificationsToShow(state) {
+  const notifications = Object.values(state.metamask.notifications) || [];
+  const notificationFilters = getNotificationFilters(state);
+  const notificationsToShow = notifications.filter(
+    (notification) =>
+      !notification.isShown && !notificationFilters[notification.id],
+  );
+  const notificationsSortedByDate = notificationsToShow.sort(
+    (a, b) => new Date(b.date) - new Date(a.date),
+  );
+  return notificationsSortedByDate;
+}
+
+export function notifcationActionFunctions(
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Was this left in accidentally? These parameters appear to be unused.

Reply:
oh, yeah

Reply:
Removed with 16ff6e67c